openapi.yaml

```diff
@@ -209,6 +203,39 @@ parameters:
     type: string
     required: true
     description: '"Bearer [token]" where [token] is your OAuth2 access token'
+  pageNumber:
+    name: page[number]
+    in: query
+    type: integer
+    required: false
+    description: Page number of results
+    default: 1
+    minimum: 1
+  pageSize:
```

Comment:
Are we using pagination parameters for this API?

Reply:
nope. Removed